Readme.md

```diff
@@ -58,8 +58,37 @@ The above command will create a VM in the `us-east1-b` zone (default) with the n
 	  --sleep_between_retry
 		                number of seconds to sleep between each retry.
 		                (default: 5)
-	  --create_delete       if you want to create/delete new VM. (default: True)
+	  --db_instance_name
+                        the name of the gcloud instance (default: envoy-db-
+                        instance)
+	  --tier            the tier of GCloud SQL service (default:
+		                db-n1-standard-2)
+	  --db_username     username on the DB (default: root)
+	  --table_name      the table which stores the benchmarking data (default:
+		                envoy_stat)
+	  --envoy_hash      the hash of envoy version (default: None)
+	  --runid           the run id of this benchmark (default: 0)
+	  --database 	    name of the database (default: envoy_stat_db)
+	  --create_delete   if you want to create/delete new benchmarking VM.
+		                (default: True)
 	  --no-create_delete
-	  --setup               if you want to run setup. (default: True)
+	  --setup           if you want to run setup on benchmarking VM. (default:
+		                True)
 	  --no-setup
-	                        
\ No newline at end of file
+	  --create_db_instance  
+	  					turn on if you want to create a Google Cloud SQL
+		                instance (default: True)
+	  --no-create_db_instance
+	  --create_db       turn on if you want to create the DB (default: True)
```

Comment:
Yeah, this parameter is for the `benchmark.py` script, not the `show_graph.py` script. I forgot to add those in the earlier PRs.